Command/QueueCreateCommand.php

```diff
@@ -73,27 +76,27 @@ protected function configure()
     }
 
     /**
-     * @inheritDoc
+     * {@inheritdoc}
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $queueName = $input->getArgument('name');
-        if ($this->getContainer()->has(sprintf('tritran.sqs_queue.%s', $queueName))) {
+        if ($this->container->has(sprintf('tritran.sqs_queue.%s', $queueName))) {
             throw new \InvalidArgumentException(sprintf('Queue [%s] exists. Please use another name.', $queueName));
         }
 
         $io = new SymfonyStyle($input, $output);
         $io->title(sprintf('Start creating a new queue which name is <comment>%s</comment>', $queueName));
 
         /** @var QueueManager $queueManager */
-        $queueManager = $this->getContainer()->get('tritran.sqs_queue.queue_manager');
+        $queueManager = $this->container->get('tritran.sqs_queue.queue_manager');
         $queueUrl = $queueManager->createQueue($queueName, [
-            'DelaySeconds' => $input->getOption('delay_seconds'),
-            'MaximumMessageSize' => $input->getOption('maximum_message_size'),
-            'MessageRetentionPeriod' => $input->getOption('message_retention_period'),
+            'DelaySeconds'                  => $input->getOption('delay_seconds'),
+            'MaximumMessageSize'            => $input->getOption('maximum_message_size'),
+            'MessageRetentionPeriod'        => $input->getOption('message_retention_period'),
             'ReceiveMessageWaitTimeSeconds' => $input->getOption('receive_message_wait_time_seconds'),
-            'VisibilityTimeout' => $input->getOption('visibility_timeout'),
-            'ContentBasedDeduplication' => $input->getOption('content_based_deduplication')
+            'VisibilityTimeout'             => $input->getOption('visibility_timeout'),
+            'ContentBasedDeduplication'     => $input->getOption('content_based_deduplication'),
```

Comment:
Dear @N-M ,
Highly appreciated your work. Just a tiny request, would you please disable aligning key-value pair in your IDE for consistent purpose? Then I could merge. Thanks in advance

Reply:
Sure I will do it. Reason why I aligned them is because that was the error that we got from styleci. Thanks

Reply:
Thanks for your contribution!